### DISCUSSION TITLE
How to configure dalli client for TLS

### DISCUSSION THREAD

**Discussion by harishsundaram17** [petergoldstein/dalli#975] [Q&A]
I am trying to connect to Memecached cluster after enabling intransit encryption. I am trying to connect through rails console but it fails to connect to server.

```
Errno::ECONNRESET: Connection reset by peer - recv

```

What is pem file [here](https://github.com/petergoldstein/dalli/wiki/Using-Dalli-with-SSL-TLS#:~:text=ssl_context.ca_file%20%3D%20%3Cpath%20to%20PEM%20file%3E)? I cannot find any .pem file in my ssl folder

**Comment by olleolleolle** [comment]
👋 Hello, PEM (privacy-enhanced mail) is a "container" file format.

You have probably already learned more about it, but here's a historical overview answer on a Q&A website: https://serverfault.com/questions/9708/what-is-a-pem-file-and-how-does-it-differ-from-other-openssl-generated-key-file
